02_repeatString/repeatString.js

```diff
@@ -1,6 +1,10 @@
-const repeatString = function() {
-
+const repeatString = function (string, number) {
```

Comment:
The argument name `number` can be too ambiguous. Something like `repeatCount` would describe the usage better.

```suggestion
const repeatString = function (string, repeatCount) {
```